app/core/logging.py

```
@@ -0,0 +1,222 @@
+            
+            Returns:
+                dict: The updated event dictionary with the "exception" field truncated to its first line, if present.
+            
+            Example:
+                >>> event = {"exception": "ValueError: Invalid input\\nTraceback (most recent call last):\\n..."}
+                >>> format_exception_short(None, None, event)
+                {'exception': 'ValueError: Invalid input'}
+            
+            Generated by CodeRabbit
+            """
+            if "exception" in event_dict and event_dict["exception"]:
```

Ruff (0.11.9)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)